CODEOWNERS

```diff
@@ -363,6 +363,10 @@
 /drivers/spi/spi_rv32m1_lpspi*            @karstenkoenig
 /drivers/spi/*esp32*                      @sylvioalves
 /drivers/spi/*pl022*                      @soburi
+/drivers/sdhc/                            @danieldegrasse
+/drivers/sdhc/sdhc_cdns*                  @roymurlidhar
+/drivers/timer/*apic*                     @dcpleung @nashif
```

Comment:
Why are these lines related to `timer` being added to codeowners?

Reply:
@roymurlidhar can you please clarify why the line regarding `timers` is being added? I don't see how it relates to the scope of this PR.